pkg/controller/ps/controller.go

```diff
@@ -1182,6 +1190,38 @@ func (r *PerconaServerMySQLReconciler) reconcileCRStatus(ctx context.Context, cr
 		cr.Status.State = cr.Status.MySQL.State
 	}
 
+	if cr.Spec.MySQL.IsGR() {
+		cli, err := clientcmd.NewClient()
+		if err != nil {
+			return err
+		}
+
+		pods, err := k8s.PodsByLabels(ctx, r.Client, mysql.MatchLabels(cr))
+		if err != nil {
+			return errors.Wrap(err, "get pods")
+		}
+
+		var outb, errb bytes.Buffer
+		cmd := []string{"/bin/bash", "-c", "cat /var/lib/mysql/full-cluster-crash"}
```

Comment:
Is there a particular reason to have `"bin/bash", "-c"` before the actual command?

Reply:
no real reason, updated